trajopt_optimizers/trajopt_sqp/src/trust_region_sqp_solver.cpp

```diff
@@ -76,6 +77,10 @@ void TrustRegionSQPSolver::solve(const QPProblem::Ptr& qp_problem)
 {
   status_ = SQPStatus::RUNNING;
 
+  // Start time
+  using Clock = std::chrono::high_resolution_clock;
```

Comment:
Use std::chrono::steady_clock instead, as usage of [high_resolution_clock](https://en.cppreference.com/w/cpp/chrono/high_resolution_clock) is discouraged (see Notes section of link).

Reply:
fixed